readme-docs/GT-CRIVO.md

```diff
@@ -19,7 +19,7 @@ docker compose build
 docker compose -f docker-compose.yml -f docker-compose-crivo.yml up crivo-init
 # Launch the containers, pass the `-d` parameter if you want to detach
 # container output from the terminal:
-docker compose up
+docker compose up --remove-orphans
```

Comment:
Let's instead try to replace line 19 with

`docker compose -f docker-compose.yml -f docker-compose-crivo.yml run --rm crivo-init`

If this doesn't work, then we can keep as-is.